### ISSUE TITLE
Event when change current location

### ISSUE THREAD

**Issue by agiletuan** [amay077/Xamarin.Forms.GoogleMaps#392]
### SUMMARY

Please don't skip my question!

### DETAILS

In other issues, amay077 help us by Xamarin Geolocator Plugin, but, if Xamarin.Forms.GoogleMaps has not support for current location, what is made Blue point on map move when i change my phone's position?

Please view on image: https://www.flickr.com/photos/145969682@N07/37336803000/in/dateposted-public/

### PLATFORMS

which platform do you want?

- [x] Android
- [x] iOS
- [ ] UWP
https://www.flickr.com/photos/145969682@N07/37336803000/in/dateposted-public/

**Comment by deextercz** [comment]
I was also interested and here is the answer:
https://stackoverflow.com/questions/22519847/what-does-map-setmylocationenabledtrue-really-do

TLDR; it is not possible in API v2
po 9. 10. 2017 v 16:55 odesílatel Nguyễn Anh Tuấn Agile <
notifications@github.com> napsal:

**Comment by amay077** [comment]
I found [GoogleMap.setLocationSource](https://developers.google.com/android/reference/com/google/android/gms/maps/GoogleMap.html#setLocationSource(com.google.android.gms.maps.LocationSource)) in Google Maps Android API that can replace the location source of the my-location layer.

But I can't found same feature in Google Maps iOS SDK.

**Comment by agiletuan** [comment]
Thank you so much!
